app/services/vot/parser.rb

```diff
@@ -25,6 +25,8 @@ def parse
         elsif acr_values.present?
           map_initial_acr_values_to_component_values
         end
+      raise 'VoT parser called without VoT or ACR values' unless initial_components
```

Comment:
The `Vot::Parser` class has it's own `ParseException` that we could raise here. Alternatively we could add an `ArgumentError` raise in the initializer where the incorrect arguments get passed int.

Reply:
Modified to throw a `ParseException`